src/index.html

```diff
@@ -4,11 +4,88 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
     <title>Stencil Component Starter</title>
-
+    <style>
+      body {
+        font-family: Arial, sans-serif;
+        margin: 70px;
+        padding: 0;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        height: 100vh;
+        background-color: #f1f1f1;
+      }
+      .checkbox-list {
+        margin-top: 20px;
+        border: 1px solid #ccc;
+        width: 300px;
+        height: auto;
+      }
+      .checkbox-list label {
+        display: block;
+        padding: 10px;
+        background-color: #f1f1f1;
+        border-bottom: 1px solid #ccc;
+        text-align: center;
+      }
+      .figma {
+        margin-top: 20px;
+        border: 1px solid #ccc;
+        width: 400px;
+      }
+      ip-checkbox-list {
+        display: block;
+        padding: 10px;
+        background-color: #f1f1f1;
+        text-align: left;
+        font-size: 12px;
+      }
+    </style>
     <script type="module" src="/build/ip-checkbox.esm.js"></script>
     <script nomodule src="/build/ip-checkbox.js"></script>
   </head>
   <body>
-    <my-component first="Stencil" last="'Don't call me a framework' JS"></my-component>
+    <div class="checkbox">
+      <ip-checkbox default-checked="true" id="check me"> Check me ! </ip-checkbox>
+    </div>
+
+    <div class="checkbox-list design">
+      <ip-checkbox-list
+        legend="Preferences:"
+        options='[
+      {"id": "1", "label": "Option 1"}, 
+      {"id": "2", "label": "Option 2"}, 
+      {"id": "3", "label": "Option 3"}
+      ]'
+      >
+      </ip-checkbox-list>
+    </div>
+
+    <div class="checkbox-list figma">
+      <ip-checkbox-list
+        legend="Select the options below:"
+        options='[
+      {"id": "agree", "label": "I agree to the terms and conditions.", "defaultChecked": true , "disabled": true},
+      {"id": "remember", "label": "Remember me on this device", "disabled": true}, 
+      {"id": "newsletter", "label": "subscribe to our newsletter for updates.","defaultChecked": true},
+      {"id": "send", "label": "Send me promotional offers and discounts."},
+      {"id": "participate", "label": "Participate in our user feddback program."}
+      ]'
+      >
+      </ip-checkbox-list>
+    </div>
+    <script>
+      const checkboxList = document.querySelector('ip-checkbox-list');
+      const checkbox = document.querySelector('ip-checkbox');
+
+      checkboxList.addEventListener('change', event => {
+        console.log(event.detail);
+      });
+
+      checkbox.addEventListener('change', event => {
+        const isChecked = event.detail;
+        console.log(isChecked);
```

Comment:
i have writted a comment about this console log on previous mr

Reply:
https://github.com/docaxess/WC-ip-checkbox/pull/1/files#r1647533112

Reply:
i already explain in readme , Event details will be displayed in the console